apps/wa/aat/base/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ resources:
   - ../../base
 patchesStrategicMerge:
   - ../../identity/aat.yaml
+  - ../../wa-task-batch-reconfiguration/aat-00.yaml
```

Comment:
if you want to patch only to aat-00 you may need to add 00 & 01 folder & ensure it is kustomized to patch both

Reply:
added aat-00 and aat-01